config/routes.rb

```diff
@@ -332,11 +332,7 @@
     namespace :ecf do
       resources :payment_breakdowns, only: [] do
         resources :statements, only: %i[show] do
-          resources :declarations, only: [] do
-            collection do
-              get :voided
-            end
-          end
+          resources :voided, controller: "participant_declarations/voided", path: "voided", only: %i[show]
```

Comment:
as this is now a resource with `show` the url now looks like this `/finance/ecf/payment_breakdowns/8cdab1e5-bed5-4cac-9821-dd93de30f3d3/statements/february-2022/voided/february-2022` which i think is undesired